packages/pagination/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@availity/pagination",
-  "version": "2.15.2",
+  "version": "2.15.3-alpha.0",
```

Comment:
QUESTION: When we create alpha versions it automatically creates a commit. Is it okay to leave this in there or does it need to be undone?

Reply:
I think it should be ok. The commit did have `[skip ci]` in the message so the PR checks are not running. We need to tweak the setup so that doesn't happen. Would you be able to edit the commit message to remove the `[skip ci]`? The checks do a dry-run of the version bump so we can confirm it will be bumped properly